packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -561,25 +568,58 @@ export abstract class FluidDataStoreContext
 		this.channel!.setConnectionState(connected, clientId);
 	}
 
-	public process(
-		message: ISequencedDocumentMessage,
+	/**
+	 * Process the messages to maintain backwards compatibility. The `processMessages` function is added to
+	 * the channel in 2.4.0. For channels before that, call `process` for each message.
+	 */
+	private processMessagesCompat(
```

Comment:
I'm not sure how to think about this name. Is the intention to keep this function in the future, even when we call channel.processMessages unconditionally? Or the intention is to inline this code and remove it later on?
If it stays, then maybe we need better name.

Reply:
It is temporary. I added comments to clarify that and created a task to clean it up.